src/lumigo_tracer/event/event_dumper.py

```diff
@@ -221,7 +235,13 @@ def dump_event(
         for handler in handlers:
             try:
                 if handler.is_supported(event):
-                    return lumigo_dumps(handler.parse(event), max_size)
+                    return lumigo_dumps(
+                        handler.parse(event),
+                        max_size,
+                        omit_skip_path=None
+                        if should_scrub_known_services()
+                        else handler.get_omit_skip_path(),
```

Comment:
probably the `if` should be inside the `lumigo_dumps` function.
It seems like anyone else that will use this function will rewrite this if, right?